css/index.css

```diff
@@ -1,3 +1,266 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container{
+	width: 880px;
+    margin: 0 auto;
+}
+
+@media( max-width: 800px ) {
+    .container {
+      width: 100%;
+    }
+  }
+
+  @media( max-width: 500px ) {
+    .container {
+      width: 100%;
+    }
+  }
+
+p {
+	line-height: 1.25;
+	margin: 20px 0;
+}
+
+nav {
+	display: inline-block;
+	margin: 20px 0;
+}
+
+nav a {
+	font-size: 15px;
+	color: grey;
+	text-decoration: none;
+	margin: 20px 20px;
+	padding-right: 20px;
+}
+
+nav a:hover {
+	text-decoration: underline;
+}
+
+.logo {
+	display: inline-block;
+	height: 40px;
+  	font-weight: bold;
+	width: 20%;
+	margin-top: 10px;
+}
+
+@media( max-width: 800px ) {
+    .navbar {
+      display: flex;
+      flex-direction: column-reverse;
+      align-items: center;
+    }
+}
```

Comment:
it would also be a good idea to give your top-nav a flex-wrap: nowrap;
and a width of percentage with 100% so that it does not overflow onto the next line in tablet view